engine/rotationmanager/advance.go

```diff
@@ -35,6 +38,10 @@ func calcAdvance(t time.Time, rot *rotation.Rotation, state rotation.State, part
 		return nil
 	}
 
+	if !newStart.After(t.Add(-15 * time.Minute)) {
+		log.Debugf(ctx, "rotation advanced late (%s)", t.Sub(newStart).String())
```

Comment:
So I've been thinking on this, and maybe this should be logged as an error -- as it does indicate a failure mode -- the wrong person was active for awhile since it's behind, and we're catching up.

The original idea with Debug was to prevent unnecessary noise, but the only case I can think of in normal operation that would trigger this would be a config change, but that behavior would be unexpected. In all others, at least in a production environment, it would indicate some type of problem with rotation processing.

Fixing the config change behavior by updating the shift start time would prevent that scenario from occurring as well.

@m17ch What do you think?

Reply:
Yeh - I think this makes sense to log as an error if it's not something we expect to see a lot and want visibility into it.